### PR TITLE
Smt12 improved incomplete registration logging

### PR DESCRIPTION
**Pull request by ikapelyukhin**

### CODE REVIEW

**Comment by ikapelyukhin** [comment]
Would be nice to merge this before the migrations PR.

File: www/perl-lib/SMT/Rest/SCCAPIv2.pm [hunk 1]
```diff
@@ -638,7 +646,8 @@ sub _registrationResult
     my $p = $self->_getProduct($product_id);
     if ( scalar(keys %{$p}) == 0)
     {
-        $self->request()->log_error("Incomplete Registration found. Unable to find the product");
+        my $guid = $self->user();
+        $self->request()->log_error("Incomplete Registration found. Unable to find the product $product_id, client ID: $guid");
```
Comment (felixsch) [inline www/perl-lib/SMT/Rest/SCCAPIv2.pm:650]: why not here also use `sprintf`?

**Review by felixsch — CHANGES_REQUESTED** [review]

File: www/perl-lib/SMT/Rest/SCCAPIv2.pm (hunk 1 above)
Comment (ikapelyukhin) [reply www/perl-lib/SMT/Rest/SCCAPIv2.pm:650]: Other strings contain method calls and hash elements. Here it's just two scalars. Why over-complicate things?

Comment (felixsch) [reply www/perl-lib/SMT/Rest/SCCAPIv2.pm:650]: Just thought the whole source code seems to use the `sprintf` thing. but ok sure its over-complicating things

Comment (felixsch) [reply www/perl-lib/SMT/Rest/SCCAPIv2.pm:650]: but anyway:

**Review by felixsch — APPROVED** [review]
LGTM!

**Pull request merged**